low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -204,23 +214,23 @@ class SSPLBundleGenerate(object):
                 post_data=f'{singleton_realstorencl.URL_ENCLLOGS_POSTDATA}/"{SUPPORT_REQUESTOR_NAME}"{SUPPORT_EMAIL_ID}"{SUPPORT_CONTACT_NUMBER}')
 
             if not response:
-                print("{0}:: {2} status unavailable as ws request {1}"
+                logger.error("{0}:: {2} status unavailable as ws request {1}"
                     " failed".format(singleton_realstorencl.LDR_R1_ENCL, url, "Debug log"))
                 break
 
             elif response.status_code != singleton_realstorencl.ws.HTTP_OK:
-                print("{0}:: http request {1} to get {3} failed with"
+                logger.debug("{0}:: http request {1} to get {3} failed with"
```

Comment:
any error should be reported with error log level

Reply:
Done